packages/demand-api/src/payment/PaymentService.ts

```diff
@@ -26,17 +35,64 @@ export class PaymentService implements Payment {
     )
   }
 
+  /** @deprecated use getBraintreeClientNonce instead */
+
   getClientNonce(params: ClientNonceParams) {
     return this.http.post<ClientNonceResponse>(
       `${this.url}/payment-methods/braintree/get-payment-method`,
       params
     )
   }
 
+  /** @deprecated use addBraintreePaymentCard instead */
+
   addPaymentCard(params: AddPaymentCardParams) {
     return this.http.post<ClientNonceResponse>(
       `${this.url}/payment-methods/braintree/add-payment-details`,
       params
     )
   }
+
+  createBraintreeClientToken(params: CreateTokenParams) {
+    return this.http.post<CreateTokenResponse>(
+      `${this.url}/payment-methods/braintree/client-tokens`,
+      {},
+      {},
+      params
+    )
+  }
+
+  getBraintreeClientNonce(params: ClientNonceParams) {
+    return this.http.post<ClientNonceResponse>(
+      `${this.url}/payment-methods/braintree/get-payment-method`,
+      params
+    )
+  }
+
+  addBraintreePaymentCard(params: AddPaymentCardParams) {
+    return this.http.post<ClientNonceResponse>(
+      `${this.url}/payment-methods/braintree/add-payment-details`,
+      params
+    )
+  }
+
+  getPaymentProvider() {
+    return this.http.get<PaymentProvidersResponse>(`${this.url}/providers`)
+  }
+
+  getOriginKey() {
+    return this.http.get<OriginKeyResponse>(`${this.url}/adyen/origin-keys`)
+  }
+
+  getPaymentMethods(params: PaymentMethodsParams) {
+    return this.http.post<PaymentMethodsResponse>(`${this.url}/adyen/payments-methods`, params)
+  }
+
+  createPaymentAuth(params: PaymentAuthParams) {
+    return this.http.post<PaymentAuthResponse>(`${this.url}/adyen/payments`, params)
+  }
+
+  getPaymentDetails(params: PaymentDetailsParams) {
+    return this.http.post<object>(`${this.url}/adyen/payments-details`, params)
+  }
```

Comment:
Maybe it's better to call this method with Adyen inside? Just to avoid the same problem as we have with Braintree now?

Reply:
Ok, change it)